scripts/update_core_lowerbound.py

```diff
@@ -1,52 +1,88 @@
+#!/bin/env python
+
+import re
```

Comment:
I don't see `re` used anywhere else. Can it be removed?

Reply:
right.